app/views/results/common/_context_menu.js.erb

```diff
@@ -0,0 +1,121 @@
+<script>
```

Comment:
I'd like to stop adding JavaScript files like this in `app/views` and putting them in `app/assets/javascripts` instead. It should just work to move it and then use a `javascript_include_tag` (I don't care about the extra request here).